geoh5py/workspace/workspace.py

```diff
@@ -15,7 +15,7 @@
 #  You should have received a copy of the GNU Lesser General Public License
 #  along with geoh5py.  If not, see <https://www.gnu.org/licenses/>.
 
-# pylint: disable=R0904
+# pylint: disable=R0904, R0913, R0914
 # pylint: disable=C0302
```

Comment:
**suggestion:** disable pylint statement with short names instead of numbers
```suggestion
# pylint: disable=too-many-lines, too-many-public-methods, too-many-arguments, too-many-locals
```